src/Microsoft.Data/Microsoft.Data.DataFrame.csproj

```diff
@@ -69,46 +98,74 @@
     </Compile>
-    <Compile Update="DataFrameBinaryOperators.cs">
+    <Compile Update="DataFrame.BinaryOperations.cs">
       <DesignTime>True</DesignTime>
       <AutoGen>True</AutoGen>
-      <DependentUpon>DataFrameBinaryOperators.tt</DependentUpon>
+      <DependentUpon>DataFrame.BinaryOperations.tt</DependentUpon>
     </Compile>
-    <Compile Update="DataFrameColumnArithmeticTemplate.cs">
+    <Compile Update="DataFrame.BinaryOperators.cs">
       <DesignTime>True</DesignTime>
       <AutoGen>True</AutoGen>
-      <DependentUpon>DataFrameColumnArithmeticTemplate.ttinclude</DependentUpon>
+      <DependentUpon>DataFrame.BinaryOperators.tt</DependentUpon>
+    </Compile>
+    <Compile Update="DataFrameBinaryOperators.cs">
+      <DesignTime>True</DesignTime>
+      <AutoGen>True</AutoGen>
+      <DependentUpon>DataFrameBinaryOperators.tt</DependentUpon>
     </Compile>
     <Compile Update="PrimitiveColumn.BinaryOperations.cs">
       <DesignTime>True</DesignTime>
       <AutoGen>True</AutoGen>
       <DependentUpon>PrimitiveColumn.BinaryOperations.tt</DependentUpon>
     </Compile>
+    <Compile Update="PrimitiveColumn.Computations.cs">
+      <DesignTime>True</DesignTime>
+      <AutoGen>True</AutoGen>
+      <DependentUpon>PrimitiveColumn.Computations.tt</DependentUpon>
+    </Compile>
     <Compile Update="PrimitiveColumnArithmetic.cs">
       <DesignTime>True</DesignTime>
       <AutoGen>True</AutoGen>
       <DependentUpon>PrimitiveColumnArithmetic.tt</DependentUpon>
     </Compile>
+    <Compile Update="PrimitiveColumnComputations.cs">
+      <DesignTime>True</DesignTime>
+      <AutoGen>True</AutoGen>
+      <DependentUpon>PrimitiveColumnComputations.tt</DependentUpon>
+    </Compile>
     <Compile Update="PrimitiveColumnContainer.BinaryOperations.cs">
       <DesignTime>True</DesignTime>
       <AutoGen>True</AutoGen>
       <DependentUpon>PrimitiveColumnContainer.BinaryOperations.tt</DependentUpon>
     </Compile>
-    <Compile Update="PrimitiveDataFrameColumnArithmetic.cs">
+    <Compile Update="PrimitiveColumnArithmetic.cs">
+      <DesignTime>True</DesignTime>
+      <AutoGen>True</AutoGen>
+      <DependentUpon>PrimitiveColumnArithmetic.tt</DependentUpon>
+    </Compile>
+    <Compile Update="PrimitiveColumnContainer.BinaryOperations.cs">
       <DesignTime>True</DesignTime>
       <AutoGen>True</AutoGen>
-      <DependentUpon>PrimitiveDataFrameColumnArithmetic.tt</DependentUpon>
+      <DependentUpon>PrimitiveColumnContainer.BinaryOperations.tt</DependentUpon>
     </Compile>
-    <Compile Update="PrimitiveDataFrameColumnContainer.BinaryOperations.cs">
+    <Compile Update="strings.Designer.cs">
       <DesignTime>True</DesignTime>
       <AutoGen>True</AutoGen>
-      <DependentUpon>PrimitiveDataFrameColumnContainer.BinaryOperations.tt</DependentUpon>
+      <DependentUpon>strings.resx</DependentUpon>
     </Compile>
   </ItemGroup>
 
+  <ItemGroup>
+    <EmbeddedResource Update="strings.resx">
```

Comment:
Any reason `strings.resx` is lower cased? Why not `Strings.resx`? It seems more usual.

Reply:
Fixed